crypto/decrypt.go

```diff
@@ -0,0 +1,65 @@
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"encoding/base64"
+	"fmt"
+	"os"
+
+	"github.com/golang/glog"
+)
+
+func pkcs7Unpad(data []byte) ([]byte, error) {
```

Comment:
I saw that the lib we're using in the other implementation does a couple more checks on the padding block, like [if the padded bytes have the right value](https://github.com/d1str0/pkcs7/blob/master/pkcs7.go#L83). Would be good to add some of those checks to this impl, if we want to avoid the external dependency for the simple logic.